packages/ui/src/index.ts

```diff
@@ -207,7 +208,7 @@ export * from './colors'
 export * from './focus'
 export * from './resize'
 
-export function createApp (target: HTMLElement): SvelteComponent {
```

Comment:
Please call rush format, before commit changes. Since linter will not pass a build.

Reply:
Ok, did